components/specificity-meter/api/src/main/java/org/phenotips/data/internal/MonarchPatientScorer.java

```diff
@@ -165,6 +165,10 @@ public double getScore(Patient patient)
         return -1;
     }
 
+    public int getScorerPriority() {
+        return 2;
```

Comment:
I would space these out, to allow something to come between monarch and omimInformationContent. Let's say 50 for this one, 10 for omimInformationContent.